### ISSUE TITLE
LinterCop.json and SubFolders

### ISSUE THREAD

**Issue by m3koenig** [StefanMaron/BusinessCentral.LinterCop#1106]
Hi,

could it be, that the "LinterCop.json" had problems if the objects have SubFolders?

## Example

Translation Info: <https://github.com/StefanMaron/BusinessCentral.LinterCop/wiki/LC0091>

### Structure

```Markdown
src
|-->ManualTest
|---->MyTestFolder
|------>MyTest.Page.al
LinterCop.json
```

The Linter will not use my settings. Only of I move the Linter in the same folder than the al file.

Is this correct? or did I misss something?

**Comment by Arthurvdv** [comment]
Hi @m3koenig,

Normally the `LinterCop.json` should be in the root of the project folder on the same level als the app.json.

![Image](https://github.com/user-attachments/assets/5925226e-9af4-49cc-a3fb-5e79dff47aec)

**Comment by Arthurvdv** [comment]
Hi, over the past months I’ve been putting a lot of time and care into the successor of BusinessCentral.LinterCop: ALCops (https://alcops.dev/). Because of that, this repository is no longer actively maintained.

I’m closing this issue, as it won’t be handled here anymore. If this is still relevant for you, please have a look at ALCops/Analyzers (https://github.com/ALCops/Analyzers).

To make the transition easier, you can also follow the LinterCop Migration Guide: https://alcops.dev/docs/lintercop-migration/

If you have any questions or run into issues during the transition, feel free to reach out through ALCops. Happy to help 🙂
